src/MediaHelper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\collabora_online;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+
+/**
+ * Service to assist with media entities.
+ */
+class MediaHelper {
```

Comment:
I can add interfaces for these classes, but we should first agree on the general direction and naming.